projects/storefrontapp-e2e-cypress/cypress/integration/regression/asm/asm.e2e-spec.ts

```diff
@@ -58,7 +59,7 @@ context('Assisted Service Module', () => {
       consent.giveConsent();
 
       cy.log('--> Stop customer emulation');
-      cy.get('cx-customer-emulation button').click();
+      cy.get('cx-customer-emulation [data-cy=logout]').click();
```

Comment:
As a convention, we shouldn't add attributes in the dom like `data-cy=logout` for the sole purpose of tests.
DOM elements retrieved with css selectors.